moderators.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Moderators"
+---
+
+A [moderator](governance.md#moderator) is a volunteer gatekeeper
+who manages contributions and discussions from [contributors](governance.md#contributor) and [users](governance.md#user). 
+[Moderators](governance.md#moderator) are vital to the health and success of R-multiverse.
+This page explains the prerequisites, selection criteria, and scope
+of the [moderator](governance.md#moderator) role.
+
+# Prerequisites
```

Comment:
Seeing this I wonder whether a section should be "Why should you do that". Or just a sentence although "[Moderators](governance.md#moderator) are vital to the health and success of R-multiverse." already mentions it somehow.